.github/workflows/release.yml

```diff
@@ -8,16 +8,19 @@ env:
   DOTNET_NOLOGO: true
 jobs:
   release:
-    runs-on: ubuntu-20.04
+    name: Windows
+    runs-on: windows-2019
```

Comment:
There's no reason the release needs Windows, which costs more Actions minutes than Linux. All of our release actions have been standardized on Linux unless there's an absolute need for Windows. (Like AdvancedInstaller in ServiceControl.)